packages/core/test/integration/query-interface/add-show-remove-constraint.test.ts

```diff
@@ -7,12 +7,17 @@ const dialect = sequelize.getDialect();
 
 describe('QueryInterface#{add,show,removeConstraint}', () => {
   describe('Without schema', () => {
+    const defaultSchema = dialect === 'db2' ? 'DB2INST1' : sequelize.dialect.getDefaultSchema();
```

Comment:
We can remove this now, right?

Reply:
Good catch. We need the constant but not the conditional check.